pitrery

```diff
@@ -3787,6 +3790,9 @@ case $action in
 				else
 					ts=$(ssh -n -- "$backup_ssh_target" "cat -- $(qw "$dir/backup_timestamp")")
 				fi
+				if [ "$?" -ne "0" ]; then
```

Comment:
code for newest and oldest are pretty close. Maybe it's time to write a function for that :) It's a good opportunity to move from this monolithic piece of code to something easier to maintain (and test).

Reply:
working on it, will post this week or next week depending on my schedule.

Reply:
après échange avec @blogh, ça va être chaud pour lui de valider sa fonction tout de suite.

@l00ptr est-ce que ça te va si on merge comme ça et @blogh propose une nouvelle PR avec la fonction plsu tard ?

Reply:
Here is a proposal for a "get_backup_timestamp".
I am not sure if it's the correct way of doing it.

I still need to do a round of tests with "USE_ISO8601_TIMESTAMPS".

Reply:
I tested with `USE_ISO8601_TIMESTAMPS` and saw no problems.

Reply:
Sorry i started a new review about your function. We can close this one :)